### ISSUE TITLE
Add RELATIVE keyword on graphical commands

### ISSUE THREAD

**Issue by spotlessmind1975** [spotlessmind1975/ugbasic#545]
It is suggested to add `RELATIVE` keyword in order to implement relative graphics instructions. The reason is to simplify the algorithm because with this instruction each call use the current graphical position (abscissa, ordinate or both) as if it was the (0,0) point.

I.e.

    PLOT 100,100,WHITE
    PLOT RELATIVE 5, 200, RED : ' Put a red pixel on 105, 200
    PLOT 100, RELATIVE 5, RED : ' Put a red pixel on 100, 205
    PLOT RELATIVE -10, RELATIVE -10, RED : ' Put a red pixel on 90,195
    PLOT RELATIVES -10, -20, RED : ' It is the same as RELATIVE -10, RELATIVE -20

The `RELATIVE` keyword will be implemented for these instructions:

    PLOT 
    POINT 
    CIRCLE
    DRAW 
    BOX 
    POLYLINE 
    ELLIPSE 
    PUT IMAGE
    GET IMAGE

**Comment by spotlessmind1975** [comment]
Published on COLDFIX v1.12.1 [rev. 20230121.1818].

**Comment by ldir-hector** [comment]
I got problem with this coldfix I typed this code : 
```
BITMAP ENABLE(160,200,16)
CLS 
'DRAW 80,100 TO 100,120
'DRAW  TO  RELATIVE -10, RELATIVE-10
'DRAW TO  RELATIVE 10, RELATIVE 10

PLOT 80,100,RGB(255,0,0)
PLOT RELATIVE -10,RELATIVE -10,RGB(0,255,0)
PLOT RELATIVE 10,RELATIVE 10,RGB(0,0,255)
```
And I got this screen : 
![image](https://user-images.githubusercontent.com/118670680/213884905-70f00ae2-7260-4ef2-967c-23e2b1412780.png)

The green point shall not have been this low on the screen, but the X is right. The blue point has its X right too, but its Y is wrong too compared to the green point.
I also got wrong drawing with the commented DRAW instructions.
